application-activedirectory-api/src/main/java/com/xwiki/activedirectory/internal/ActiveDirectorySetupListener.java

```diff
@@ -116,7 +116,7 @@ public void dispose() throws ComponentLifecycleException
 
     private void setAuthService(XWiki xwiki)
     {
-        xwiki.setAuthService(new ActiveDirectoryAuthServiceImpl());
+        xwiki.setAuthService(new ActiveDirectoryAuthServiceImpl(xwiki.getAuthService()));
```

Comment:
Note that this still makes the AD authenticator a very bad citizen in terms of authentication, since it makes it impossible to use another one when it's installed. Would be nice to use this opportunity to change a bit the logic.